readium/navigator/src/main/assets/_scripts/src/decorator.js

```diff
@@ -203,36 +206,61 @@ export function DecorationGroup(groupId, groupName) {
         )
       ) || 1;
     const pageSize =
-      (verticalRL ? viewportHeight : viewportWidth) / columnCount;
+      (documentWritingMode ? viewportHeight : viewportWidth) / columnCount;
```

Comment:
Isn't this supposed to be `isVertical` here? `writing-mode` could be set to an horizontal mode.

```suggestion
      (isVertical ? viewportHeight : viewportWidth) / columnCount;
```

Reply:
Good catch